### DISCUSSION TITLE
New @rnx-kit/align-deps profil for react-native@0.72.4

### DISCUSSION THREAD

**Discussion by Leroy-design** [microsoft/rnx-kit#2661] [Q&A] [answered]
### What happened?

Hey, 

Where I can find the list of supported version of React native for the align-deps library. 

Did the 0.72.4 profile is planned? 

Thank you 
Swann

### Affected Package

@rnx-kit/align-deps

### Version

npx version

### Which platforms are you seeing this issue on?

- [ ] Android
- [ ] iOS
- [X] macOS
- [ ] Windows

### System Information

```true
System:
  OS: macOS 13.5.2
  CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
  Memory: 4.23 GB / 32.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 18.15.0
    path: ~/.nvm/versions/node/v18.15.0/bin/node
  Yarn:
    version: 1.22.19
    path: ~/.nvm/versions/node/v18.15.0/bin/yarn
  npm:
    version: 9.5.0
    path: ~/.nvm/versions/node/v18.15.0/bin/npm
  Watchman:
    version: 2022.09.19.00
    path: /usr/local/bin/watchman
Managers:
  CocoaPods:
    version: 1.11.3
    path: /usr/local/bin/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 22.4
      - iOS 16.4
      - macOS 13.3
      - tvOS 16.4
      - watchOS 9.4
  Android SDK: Not Found
IDEs:
  Android Studio: 2022.1 AI-221.6008.13.2211.9619390
  Xcode:
    version: 14.3.1/14E300c
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 11.0.17
    path: /usr/bin/javac
  Ruby:
    version: 2.6.10
    path: /usr/bin/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.4
    wanted: 0.72.4
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
```

### Steps to Reproduce

npx @rnx-kit/align-deps --requirements react-native@0.72

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tido64** [comment, marked as answer]
If you check our README, under [Capabilities](https://github.com/microsoft/rnx-kit/tree/main/packages/align-deps#capabilities), there's a toggle:

![image](https://github.com/microsoft/rnx-kit/assets/4123478/28ed98ec-e0b2-4bce-8f1e-6874b1739f1f)

If you click on it, you'll get a table:

![image](https://github.com/microsoft/rnx-kit/assets/4123478/99cf2a2a-a233-4c16-99b8-cb8b2233617f)

Support for 0.72 has been out for [some time](https://github.com/microsoft/rnx-kit/pull/2322) already. We're currently waiting for the next version.
